tests/UserActionTest.php

```diff
@@ -206,15 +206,15 @@ public function testDisabled3()
 
     public function testFields()
     {
-        $client = new UaClient($this->pers);
-        $a = $client->addUserAction('change_details', ['callback' => 'save', 'fields' => ['name']]);
+        $m_client = new UaClient($this->pers);
+        $a = $m_client->addUserAction('change_details', ['callback' => 'save', 'fields' => ['name']]);
 
-        $client = $client->load(1);
+        $e_client = $m_client->load(1);
```

Comment:
Please keep changes mininal, this is not consistent style with the other changes.

I will fix this, but please do not add more unneded refactoring changes.